robottelo/ui/locators.py

```diff
@@ -672,6 +672,13 @@ def __iter__(self):
     "entity_checkbox": (
         By.XPATH,
         "//label[normalize-space(.)='%s']/input[@type='checkbox']"),
+    "entity_select_list": (
+        By.XPATH,
+        "//ul/li/div[normalize-space(.)='%s']"),
+    "select_list_search_box": (
+        By.XPATH,
+        ("//div[contains(@class, 'searchbox') and contains(@id, 'drop')]"
+         "/div/input")),
```

Comment:
Hmm, that locator gives me 5 matches right away for user entity while my locator returns back only one, unique element